### DISCUSSION TITLE
How can I inactivate an account after someone died?

### DISCUSSION THREAD

**Discussion by Dayof** [community/community#22439] [New to GitHub]
<p>A friend of mine died in 2015, he had some open sources repositories that some people now and then ask for maintenance (<a href="https://github.com/alexdantas/kure-music-player" class="inline-onebox" rel="noopener nofollow ugc">GitHub - alexdantas/kure-music-player: Open-source music player for Android</a>). How can I disable his account? And how can I close his repos?</p>

**Comment by Zerotask** [comment, marked as answer]
<p>Contact GitHub directly: <a href="https://support.github.com/contact/feedback" class="inline-onebox" rel="noopener nofollow ugc">Share feedback - GitHub Support</a></p>
<p>And you should provide a proof of it.</p>

**Comment by billythekid** [comment]
<p>Hi <a class="mention" href="/dayof">@dayof</a></p>
<p>You can read more at our <a href="https://docs.github.com/github/site-policy/github-deceased-user-policy">Deceased User Policy</a> where there’s some information with the sorts of documentation we’ll need etc.</p>
<p>Sorry for your loss.</p>
